src/components/Pickers/BasePicker.Props.ts

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { IPickerItemProps } from './PickerItem.Props';
+
+export interface IBasePickerProps extends React.Props<any> {
+  /**
+   *
+   */
+  onRenderItem?: (item: IPickerItemProps) => JSX.Element;
```

Comment:
minor nit; the param here for onRenderX methods should always be named props, and probably should take an optional defaultRender callback if applicable (may not be here.)